packages/cli/templates/default-app/plugins/welcome/dev/index.tsx

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
I don't think we want these notice headers in the templates

Reply:
Fixed!